src/nvme/log.h

```diff
@@ -59,21 +71,24 @@ int nvme_get_logging_level(nvme_root_t r, bool *log_pid, bool *log_tstamp);
  * will be set as well. This means the global root object is always pointing to
  * the latest created root object. Note the first @nvme_free_tree call will reset
  * the global root object.
+ *
+ * This funciton is depricated. Use nvme_init_default_logging or/and
+ * nvme_init_logging instead.
  */
 void nvme_set_root(nvme_root_t r);
 
 /**
  * nvme_set_debug - Set NVMe command debugging output
  * @debug:	true to enable or false to disable
  *
- * Don't use it, it's debricated.
+ * This function is depricated. Use nvme_init_default_logging instead.
```

Comment:
_depricated_ -> _deprecated_

Reply:
What about adding `__attribute__((deprecated))` annotation for deprecated symbols?